Directory.Build.props

```diff
@@ -33,6 +33,11 @@
 
     <DashboardPublishedArtifactsOutputDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'DashboardArtifacts', '$(Configuration)'))</DashboardPublishedArtifactsOutputDir>
     <WorkloadsPackageSource>$(ArtifactsShippingPackagesDir)</WorkloadsPackageSource>
+
+    <!-- xUnit2031: Do not use Where clause with Assert.Single -->
```

Comment:
Why are we disabling this one?

Reply:
@radical I wanted to avoid extra changes in this PR to keep it minimal. But I think it can be addressed later. I see you are already tracking it now in the follow-up issue.